community/kernel/src/main/java/org/neo4j/kernel/impl/locking/Locks.java

```diff
@@ -126,4 +126,43 @@ void visit( ResourceType resourceType, long resourceId, String description, long
     void accept(Visitor visitor);
 
     void close();
+
+    interface Tracer
```

Comment:
I always think of the names of nested types as being prefixed with the name of the outer type, making the name here `Locks.Tracer`, but when I wrote it I fully anticipated it to be moved to the top level and renamed `LocksTracer` at some point. The main thing that stopped me is the package structure mess.